### ISSUE TITLE
FrameworkBundle workflows enabled configuration error

### ISSUE THREAD

**Issue by sandermarechal** [symfony/symfony#28662]
**Symfony version(s) affected**: 4.1.5

**Description**  
The workflows configuration does not work as documented by `condole config:dump-reference` or `console debug:config`. The following configuration should be valid:

```yml
framework:
  workflows:
    enabled: true # explicitly enable workflows
    workflows:
      my_workflow:
        # ... workflow configuration
```
However, this configuration leads to the following error:

```
Unrecognized option "my_workflow" under "framework.workflows.workflows.workflows"
```

**How to reproduce**  

Use the configuration above on a clean Symfony, then clear the cache.

**Possible Solution**  
The bug seems to be caused in `beforeNormalization()` call for the workflow section (`src/Bundle/FrameworkBundle/DependencyInjection/Configuration.php` lines 218-236). When an `enabled` key is present, it deletes the key, then nests the rest of the configuration in an additional `workflows` key. This causes the workflows of the example config above to become invalid:

```yml
framework:
  workflows:
    enabled: true
    workflows:
      workflows:
        my_workflow:
          # ... workflow configuration
```

**Comment by lyrixx** [comment]
Hello.

Indeed this seems a bug in the ways symfony normalize the configuration.

Are you OK to update your configuration in order to use the "regular" way: 

```
framework:
    workflows:
        enabled: true # explicitly enable workflows
        verification:
            type: state_machine
            marking_store:
                type: single_state
                arguments:
                    - 'state'
            supports:
                - App\Entity\Task
            audit_trail: true
            initial_place: created
            places:
                - draft
                - created
                - pending
                - verified
                - cancelled
                - refused
            transitions:
                pending:
                    from: [created]
                    to: pending
                verified:
                    from: [pending]
                    to: verified
                cancelled:
                    from: [created, pending, verified]
                    to: cancelled
                refused:
                    from: [pending]
                    to: refused
```

If so, you can safely close this issue

**Comment by sandermarechal** [comment]
> Are you OK to update your configuration in order to use the "regular" way
I was merging and prepending several different configurations, which led to this problem. IMHO the normalization bug should be solved anyway.

**Comment by lyrixx** [comment]
@sandermarechal Thanks for your report. This has been fixed by #28769
